hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClientWriteHandler.java

```diff
@@ -91,7 +115,8 @@ private void writeBuffer() throws IOException {
         }
 
         buffer.flip();
-        socketChannel.write(buffer);
+        int bytesWritten = socketChannel.write(buffer);
```

Comment:
I'm not a 100% sure, but this could create a "hidden field" issue in SonarQube. In another place you just wrote `messagesRead.inc(builder.onData(buffer));`. I would use the same style in both places, so preferably the one without the "hidden field" issue :)